lib/fuse.c

```diff
@@ -4421,6 +4421,7 @@ static const struct fuse_opt fuse_lib_opts[] = {
 	FUSE_LIB_OPT("noforget",              remember, -1),
 	FUSE_LIB_OPT("remember=%u",           remember, 0),
 	FUSE_LIB_OPT("modules=%s",	      modules, 0),
+	FUSE_LIB_OPT("max_idle_threads=%u",   max_idle_threads, 0),
```

Comment:
You have defined `max_idle_threads` as `int`, so that this should be `%d` rather than `%u` (or you could declare `max_idle_threads` as `unsigned`, which may make more sense).